src/lib/reasoners/bitv_rel.ml

```diff
@@ -131,7 +127,12 @@ module Domains : sig
 end = struct
   type t =
     { bitlists : Bitlist.t MX.t
-    (** Mapping from semantic values to their bitlist domain. *)
+    (** Mapping from semantic values to their bitlist domain.
+
+        Note: this mapping only contains domain for leaves (i.e. uninterpreted
+        terms or AC symbols); domains associated with more complex semantic
+        values are rebuilt on-the-fly using the structure of said semantic
+        values. *)
```

Comment:
It seems this PR is rebased on #1004, which means we cannot merge it before solving #1005 too.

Reply:
It is explicitly mentioned in the PR description that it includes #1004 :)